apps/fluent-tester/src/testPages.ts

```diff
@@ -114,6 +115,12 @@ export const tests: TestDescription[] = [
     testPageButton: HOMEPAGE_NATIVEDATEPICKER_BUTTON,
     platforms: ['ios'],
   },
+  {
+    name: 'Divider',
+    component: DividerTest,
+    testPageButton: HOMEPAGE_DIVIDER_BUTTON,
+    platforms: ['win32'],
```

Comment:
Any reason we can't just put it on all the platforms?

Reply:
So far, I've only tested how the divider renders on win32, but this is expected to be on all platforms that support separator.

Reply:
If there are no objections, I'd prefer enabling it for all platforms and using the platform status section to say it's only designed for win32 at the moment if you have to.